### PR TITLE
Add align to tslint.json and fix errors.

### PR DESCRIPTION
**Pull request by ry**

### CODE REVIEW

File: tools/test_browser.ts [hunk 1]
```diff
@@ -72,17 +72,17 @@ if (!filterExpr) {
...
     timeout: 10 * 1000,
   },
-  {
+                        {
     path: "notebook/",
     doneMsg: /Propel onload/,
     timeout: 10 * 1000,
   },
-  {
+                        {
```
Comment (piscisaureus) [inline tools/test_browser.ts]: This looks pretty wonky...

**Comment by ry** [comment]
Dang - I totally missed that.

File: tools/test_browser.ts (hunk 1 above)
Comment (ry) [reply tools/test_browser.ts]: Dang - I totally missed that.

Comment (ry) [reply tools/test_browser.ts]: Fixed. PTAL.

**Pull request merged**